docs/storage/noql.adoc

```diff
@@ -283,6 +283,49 @@ fulltext('data.*', '"gone fishing today"', "AND")
 fulltext('data.*', '"gone fishing today"~2', "AND")
 ----
 
+[#stemmed]
+==== stemmed()
+
+The stemmed function is similar to <<#fulltext,fulltext()>> except that it searches language optimized tokens instead of a source text. E.g. source text `The monkey loved bananas` will be transformed to `the`, `monkey`, `love`, `banana` tokens and they will be used for search.
+
+NOTE: Stemming is language-dependant so content needs a language to be set for stemming to work !
+
+.Function definition
+[source,sql]
+----
+stemmed(<fields>, <search-string>, <operator>, <language>)
+----
+
+fields:: Comma-separated list of <<propertyPath, propertyPaths>> to include in the search.
+
+NOTE: Only `_alltext` field is currently indexed for stemming
+
+search-string:: The search-string supports a set of operators
+
+* ``+`` signifies AND operation.
+* ``|`` signifies OR operation.
+* ``-`` negates a single token.
+* ``*`` at the end of a term signifies a prefix query.
+* ``(`` and ``)`` signify precedence.
+* ``"`` and ``"`` wraps a number of tokens to signify a phrase for searching
+* ``~N`` after a word signifies edit distance (fuzziness) with a number representing https://en.wikipedia.org/wiki/Levenshtein_distance[Levenshtein distance].
+* ``~N`` after a phrase signifies slop amount (how far apart terms in phrase are allowed)
+
+
+operator:: Allowed operators are:
+
+* `OR` Any of the words in the search-string matches.
+* `AND` All words in search-string matches.
+
+language:: Content language that was used for stemming. List of <<indexing#languages, supported languages>>
```

Comment:
In previous examples, and in my test code I'm using 'yes' as the fourth parameter. Are you certain you can set stemming language as the fourth parameter? I'm really happy if that is the case because then I can run both norwegian and english in the same query with higher boost on the norwegian.

Reply:
Absolutely positive about fourth parameter !
If you use 'yes', it will be considered as language and there will be no results, because there is no 'yes' language analyzer with closest match being 'es' for spanish :)

Here is the test I used:
`stemmed('_alltext', 'monkey banana', 'OR', 'en')`

**NB:** you can only search the `_alltext` field with stemmed function!